scripts/kafka_bridge_config_generator.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+SECURITY_PROTOCOL=PLAINTEXT
+
+if [ "$KAFKA_BRIDGE_TLS" = "true" ]; then
+    SECURITY_PROTOCOL="SSL"
+
+    if [ -n "$KAFKA_BRIDGE_TRUSTED_CERTS" ]; then
+        TLS_CONFIGURATION=$(cat <<EOF
+# TLS / SSL
+ssl.truststore.location=/tmp/kafka/bridge.truststore.p12
+ssl.truststore.password=${CERTS_STORE_PASSWORD}
+ssl.truststore.type=PKCS12
+EOF
+)
+    fi
+
+    if [ -n "$KAFKA_BRIDGE_TLS_AUTH_CERT" ]; then
+        TLS_AUTH_CONFIGURATION=$(cat <<EOF
+ssl.keystore.location=/tmp/kafka/bridge.keystore.p12
+ssl.keystore.password=${CERTS_STORE_PASSWORD}
+ssl.keystore.type=PKCS12
+EOF
+)
+    fi
+fi
+
+if [ -n "$KAFKA_BRIDGE_SASL_USERNAME" ] && [ -n "$KAFKA_BRIDGE_SASL_PASSWORD_FILE" ]; then
+    if [ "$SECURITY_PROTOCOL" = "SSL" ]; then
+        SECURITY_PROTOCOL="SASL_SSL"
+    else
+        SECURITY_PROTOCOL="SASL_PLAINTEXT"
+    fi
+
+    PASSWORD=$(cat /opt/kafka/bridge-password/$KAFKA_BRIDGE_SASL_PASSWORD_FILE)
+
+    if [ "x$KAFKA_BRIDGE_SASL_MECHANISM" = "xplain" ]; then
+        SASL_MECHANISM="PLAIN"
+        JAAS_SECURITY_MODULE="plain.PlainLoginModule"
+    elif [ "x$KAFKA_BRIDGE_SASL_MECHANISM" = "xscram-sha-512" ]; then
+        SASL_MECHANISM="SCRAM-SHA-512"
+        JAAS_SECURITY_MODULE="scram.ScramLoginModule"
+    fi
+
+    SASL_AUTH_CONFIGURATION=$(cat <<EOF
+sasl.mechanism=${SASL_MECHANISM}
+sasl.jaas.config=org.apache.kafka.common.security.${JAAS_SECURITY_MODULE} required username="${KAFKA_BRIDGE_SASL_USERNAME}" password="${PASSWORD}";
+EOF
+)
+fi
+
+# Write the config file
+cat <<EOF
+#Apache Kafka common
+kafka.bootstrapServers=${KAFKA_BRIDGE_BOOTSTRAP_SERVERS}
+
+#Apache Kafka producer
+kafka.producer.acks=${KAFKA_BRIDGE_PRODUCER_ACKS}
+
+#Apache Kafka consumer
+kafka.consumer.autoOffsetReset=${KAFKA_BRIDGE_CONSUMER_AUTO_OFFSET_RESET}
```

Comment:
we don't need all the three above configuration parameter, we should get configuration for consumer and producer from the CRD where `consumer.config` and `producer.config` are now needed. @scholzj wdyt?

Reply:
I don't particularly like having two configurations ... but it sounds logical. We can also have only one set and paste it into the configuration twice with different prefixes. Kafka clients usually handle well this kind of tricks (Connect does it all the time). And I'm not sure whether there is a real need to configure the same option differently for producer and consumer. But it sould be more in sync with what we have for RHEL now.

Reply:
After discussion related to my PR #207 , it's better having common properties like bootstrap servers, security and auth related prefixed with `kafka.` while the specific one for producer and consumer with `kafka.producer.` and `kafka.producer.`. @stanlyDoge FYI